test/node/src/with-schema.test.ts

```diff
@@ -517,6 +520,72 @@ for (const dialect of DIALECTS.filter(
       })
     })
 
+    describe('raw sql', () => {
```

Comment:
I think a little copy-pasta is better than these DRY tests that are really really hard to read. This is fine, but in the future, if there's only a couple of cases, just copy paste them.

Reply:
Absolutely agree, it's a mess.